app/views/setup/discovery.html.slim

```diff
@@ -0,0 +1,34 @@
+.alert.alert-info.alert-dismissible role="alert"
+  button.close[type="button" data-dismiss="alert" aria-label="Close"]
```

Comment:
this could then be a flash message, but i think even that is broken atm, i'd say lets fix it in a follow up. the PR is already big enough again